assets/src/features/home-page/components/VideoroomHomePage.tsx

```diff
@@ -60,56 +50,67 @@ const VideoroomHomePage: React.FC = () => {
 
   return (
     <HomePageLayout>
-      <section className="flex flex-col items-center gap-y-12 sm:gap-y-18">
+      <section className="flex flex-col items-center gap-y-8 sm:justify-center sm:gap-y-14 2xl:gap-y-28">
         {deviceManager.errorMessage && (
           <div className="w-full bg-red-700 p-1 text-white">{deviceManager.errorMessage}</div>
         )}
         <div className="flex flex-col items-center gap-y-6 text-center">
-          <h2 className="text-3xl sm:text-5xl">Videoconferencing for everyone</h2>
+          <h2 className="text-3xl font-medium sm:text-5xl">Videoconferencing for everyone</h2>
```

Comment:
```suggestion
          <h2 className="text-3xl font-medium sm:text-5xl tracking-wide">Videoconferencing for everyone</h2>
```

What about a little wider kerning? 😄